src/Lime.Sample.AspNetCore/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Lime.Protocol;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Lime.Sample.AspNetCore.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ValuesController : ControllerBase
```

Comment:
I think you forgot to delete this file?

Reply:
Nope, it is here to illustrate the usage of the ASP.NET MVC middleware along with Lime.